src/core/connection_pool.c

```diff
@@ -0,0 +1,521 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    Definitions for the MsQuic Connection Pool API, which allows clients to
+    create a pool of connections that are spread across RSS cores.
+
+--*/
```

Comment:
Code coverage is still claiming all the lines in this file are uncovered...

Reply:
I believe this is because code coverage doesn't run with xdp. Probably should change that...

Reply:
I'll make a separate PR to do that. In the meantime, when reviewing, you can go into the upper right `...` menu and uncheck "Show Annotations". 
Just remember to uncheck that once you're done reviewing this PR :)